thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/events/Event.java

```diff
@@ -0,0 +1,54 @@
+package com.linkedin.thirdeye.anomaly.events;
+
+import java.util.List;
+import java.util.Map;
+
+public abstract class Event {
+  String metric;
+  String service;
+
+  long startTime;
+  long endTime;
+
+  Map<String, List<String>> targetDimensionMap;
```

Comment:
Could you give an example of targetDimensionMap? It would be helpful to understand how to use this variable.